Lesson9/Lesson9_1.cpp

```diff
@@ -0,0 +1,50 @@
+// Lesson 9 Issue 1
+#include <iostream>
+
+void translateArray(int numbers[], int size){
+    for(int i = 0; i < size; i++){
+        if(numbers[i] > 0){
+            std::cout << numbers[i] * 2 << " ";
+        }else {
+            std::cout << "0 ";
+        }
+    }
+}
```

Comment:
not translated output but modification of original array

Reply:
fixed, find some additional bugs)
and have a question for next lesson